### PR TITLE
Create data retention policy april 2016

### PR DESCRIPTION
**Pull request by samnissen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +163,21 @@ This is automatically engaged when you run `rake all:start`.
...
+This is done from an environment variable, set like so in `~/.bash_profile`:
+
+```bash
+$ export POLARISCOPE_ALLOWED_RUN_DATE_RANGE='6.months.ago'
+```
+
+The code must evaluate to a ActiveSupport::TimeWithZone object.
+Once the date is passed by a given Run, the model prune the oldest records,
+ordering by `created_at` date. If no limit is set, the pruning
+will happen at 6 months.
```
Comment (samnissen) [inline README.md]: Add instructions for acceptable date ranges

File: app/models/run.rb [hunk 2]
```diff
@@ -14,7 +17,32 @@ class Run < ActiveRecord::Base
   validates :environment, presence: true
   validates :browsers, presence: true
 
+  after_create do
+    self.class.prune(ENV['POLARISCOPE_ALLOWED_RUN_DATE_RANGE'])
```
Comment (samnissen) [inline app/models/run.rb]: Too computationally expensive. Refactor run ~ once per day

File: lib/tasks/prune.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    Run.prune(run_range)
+
+    schedule_next_backup_job
+  end
+
+  def schedule_next_backup_job
+    work
+  end
+
+  handle_asynchronously :schedule_next_backup_job, :run_at => Proc.new { 24.hours.from_now }, :queue => 'queue'
```
Comment (jhulme) [inline lib/tasks/prune.rb:22]: Is it worth moving schedule of the next job to something like task :after_prune?

Comment (samnissen) [inline lib/tasks/prune.rb:22]: Decoupling this conversation from this branch per discussion.

**Pull request merged**